src/radio/Radio.ts

```diff
@@ -29,32 +31,23 @@ import * as css from './styles/radio.m.css';
  * @property onTouchEnd       Called on the input's touchend event
  * @property onTouchCancel    Called on the input's touchcancel event
  */
-export interface RadioProperties extends ThemedProperties {
+export interface RadioProperties extends ThemedProperties, LabeledProperties, InputEventProperties {
 	checked?: boolean;
 	describedBy?: string;
 	disabled?: boolean;
 	invalid?: boolean;
-	label?: string | LabelOptions;
 	name?: string;
 	readOnly?: boolean;
 	required?: boolean;
```

Comment:
Why aren't all InputProperties removed from this interface?

Reply:
the interfaces still need some more work to split out better

Reply:
Circling back on this. Why can't we remove all of these properties yet? What additional work will be required, and will it be part of this PR?